basics/5 terminal/index.md

```diff
@@ -1,27 +1,147 @@
+
+> TODO: insert a screenshot when I have access to a Windows machine.
 
 ## Using `cd` to get to the right spot
+When working in a Terminal, you're always working within a certain folder on your device. All the commands you run will run relative to that folder. The selected folder in a Terminal window is called the 'present working directory'. To find out what folder you have selected, simply type the shorthand for that: `pwd`, and press enter.
+
+![](pwd.png)
+
+As you can see, it has just dropped me into my user directory when I opened the window. It is really unlikely I'd want to do anything here, so let's navigate elsewhere. To find out what's in our current directory, run the command `ls`.
+
+![](ls.png)
+
+As you can see, there are a lot of folders and files in my user folder. Say I want to run some code stored on my Desktop: we need to navigate there. To do so, run `cd Desktop`. If Desktop isn't one of the folders listed for you: don't worry. Just navigate to any other folder by replacing `Desktop` with the name of that folder.
+
+![](desktop.png)
+
+You can now see next to the `%` that I'm at the Desktop. Try running `pwd` to confirm this. Try to find out what folders are located here...
+
+Do notice, however, that any commands you run now are run relative to the Desktop. You cannot, for example, navigate to the Downloads folder by typing `cd Downloads`, as Downloads isn't on the Desktop. You can do so however by typing out the full path: `cd /Users/[yourname]/Downloads` or `cd ~/Downloads` (the `~` always refers to your user directory).
+
+![](downloads.png)
+
+In addition, you can use `..` to refer to one directory higher than your current directory. So when on the Desktop, you could use `cd ..` to return to your user directory. Or even better: use `cd ../Downloads` to navigate directly to Downloads.
+
+> ### Pro tip!
+> Don't feel like typing out a long path to a directory? Simply drag the folder onto your Terminal from your Finder or Explorer. The full path will apear right away!
 
 ## Creating a new repo on Github and cloning
+Now, let's get started using git. Git is a great tool for managing programming projects, from small projects to enterprise-level applications. It keeps track of all changes, and allows for you to work on a project with a team of people, all while avoiding conlicts as much as possible.
+
+Your code in git is placed in a so-called 'repository'. Within a repository, git keeps track of changes and resolved conflicts. Every repository needs to be hosted somehwere. Ofcourse, you could host it on your own device. If you're going to work with other people, however, this won't work. Most repository's, therefore, are primarily hosted through services like [Github](https://github.com). You should already have an account there. If not, create one now!
```

Comment:
repository’s -> repositories

Hiervoor geldt enigszins ook dat het in het college al benoemd is. 

Leg duidelijk uit waarom je denkt dat ze al een account hebben (in welke context hebben ze die gemaakt).